internal/services/dashboard/dashboard_grafana_resource.go

```diff
@@ -116,6 +117,16 @@ func (r DashboardGrafanaResource) Arguments() map[string]*pluginsdk.Schema {
 			Default:  true,
 		},
 
+		"grafana_major_version": {
+			Type:     pluginsdk.TypeString,
+			Optional: true,
+			ForceNew: true,
+			Default:  "9",
```

Comment:
Whilst this default value exists today, it's likely to change in the future - can we add a TODO to make this field required in 4.0, so that we don't end up needing to introduce a breaking change in the future?

```suggestion
			// TODO: make this field Required (with no default) in 4.0
			Optional: true,
			ForceNew: true,
			Default:  "9",
```